### ISSUE TITLE
[FEATURE REQUEST] Property slider view default unit

### ISSUE THREAD

**Issue by FrciSmrci** [artf/grapesjs#1346]
Hey @artf ,

I think that the slider component would come handy from the UX perspective for the user interaction of some of the integer properties as well, like font-size / line-height, etc. 

I would like to propose that the slider component is extended with a default unit property which would take in an string, but without the selection array under units property.  

What do you think?

Thank you in advance!

**Comment by artf** [comment]
It could be already possible with this kind of configuration
```js
{
        type: 'slider',
        property: 'font-size',
        units: [],
        unit: 'em',
        defaults: 14,
        max: 40,
        min: 1,
}
```
but unfortunately I notice that currently the slider itself does not update its pointer. I'll fix that in the next release

**Comment by FrciSmrci** [comment]
Thank you @artf :)

**Comment by FrciSmrci** [comment]
@artf Not sure if this question slipped by.

**Comment by artf** [comment]
Yeah, sorry, I was too much busy with other stuff this week, hope to release this weekend 😬

**Comment by FrciSmrci** [comment]
@artf Awesome, thank you! :)

**Comment by FrciSmrci** [comment]
Hey @artf , thanks for the enhancement! The implementation works as expected with some properties (font-size, border-radius, ...), but not with others (line-height, border-width, ...). I created a [example/demo](https://stackblitz.com/edit/angular-737uyd) where it's visible that the configuration is the same in both scenarios, however the unit doesn't apply in the case of previous mentioned properties. 

Thank you.
